via/CMakeLists.txt

```diff
@@ -66,16 +66,27 @@ elseif (MSVC)
     target_compile_definitions(vkvia PRIVATE _CRT_SECURE_NO_WARNINGS _USE_MATH_DEFINES)
 endif()
 
-# https://github.com/LunarG/VulkanTools/issues/1908
-find_package(PkgConfig)
-if (PKG_CONFIG_FOUND)
-    pkg_check_modules(jsoncpp REQUIRED IMPORTED_TARGET jsoncpp)
-    target_link_libraries(vkvia PRIVATE PkgConfig::jsoncpp)
-else()
-    find_package(jsoncpp CONFIG REQUIRED)
+# NOTE: jsoncpp hasn't had updates in a bit and isn't getting fixes.
+# may be worth using nlohmann/json in the future which gets more updates
+# and has better CMake support.
+find_package(jsoncpp CONFIG)
+if (TARGET jsoncpp_static)
```

Comment:
Not because of a bug in jsoncpp I can't detect if the find_package worked like with other packages so I have to detect the target instead.